packages/features/schedules/components/Schedule.tsx

```diff
@@ -241,10 +240,10 @@ const RemoveTimeButton = ({
   );
 };
 
-const TimeRangeField = ({ className, value, onChange }: { className?: string } & ControllerRenderProps) => {
+const TimeRangeField = ({ value, onChange }: ControllerRenderProps) => {
   // this is a controlled component anyway given it uses LazySelect, so keep it RHF agnostic.
   return (
-    <div className={className}>
```

Comment:
why did you remove className? 
you can do 
<div className={classNames("flex flex-row gap-1",className)}>

Reply:
Ahh that's true. I will fix it.Thank you

Reply:
@Udit-takkar Done.